tests/system/Database/Builder/SelectTest.php

```diff
@@ -105,7 +105,7 @@ public function testSelectWorksWithRawSql()
     {
         $builder = new BaseBuilder('users', $this->db);
 
-        $sql = 'REGEXP_SUBSTR(ral_anno,"[0-9]{1,2}([,.][0-9]{1,3})([,.][0-9]{1,3})") AS ral';
+        $sql = 'REGEXP_SUBSTR(ral_anno,"\d{1,2}([,.]\d{1,3})([,.]\d{1,3})") AS ral';
```

Comment:
Why does rector change this line? This is just a string.

Or if `\d` in `REGEXP_SUBSTR` (Oracle function) matches unicode number characters like `६`, this is a breaking change.

Reply:
Ok, I will remove the rule from the list.

Reply:
It seems rector just replaces string `[0-9]`.
I created an issue: https://github.com/rectorphp/rector/issues/7993